templates/org/aac/aac-article-title-content.md

```diff
@@ -10,132 +11,148 @@ _Download a [Visio file](https://arch-center.azureedge.net/architecture.vsdx) th
 
 ### Workflow
 
-> Use a numbered list if there are corresponding numbers in the diagram. If not, use a bulleted list.
+<!--
+- Title this section "Workflow" if data isn't in the scenario.
+- Use a numbered list that describes the dataflow or workflow of each step in the solution. Use a bulleted list if there are no numbers in the diagram. Start from the user or external data source, and then follow the flow through the rest of the solution. The following section uses the previous diagram as an example and should be updated for your specific article. The following dataflow corresponds to the previous diagram:
+-->
 
-The following workflow (or dataflow) corresponds to the above diagram:
+The following workflow (or dataflow) corresponds to the previous diagram:
 
 - **Thing 1**. Explain what happens with the first technology in the diagram.
 
 - **Thing 2**. Explain what happens with the second technology in the diagram.
 
 ### Components
-
-> A bulleted list of components in the architecture (including all relevant Azure services) with links to the product service pages. This is for lead generation (what business, marketing, and PG want). It helps drive revenue.
-
-> Why is each component there?
-> What does it do and why was it necessary?
-> Link the name of the service (via embedded link) to the service's product service page. Be sure to exclude the localization part of the URL (such as "en-US/").
-
-- Examples: 
-  - [Azure App Service](https://azure.microsoft.com/services/app-service)
-  - [Azure Bot Service](https://azure.microsoft.com/services/bot-service)
+<!-- 
+- Add a bulleted list of components in the architecture. It includes all relevant Azure services and has links to the Well-Architected Framework service guide for the product. This is for lead generation (what business, marketing, and PG want). It helps drive revenue.
+- Describe why each component is necessary and what it does.
+- Link the name of the service via embedded link to the Azure Well-Architected service guide if it exists, or to the service's product page. Exclude the localization part of the URL, such as `en-us`.
+- The following section contains example components but should be updated for your specific article.
+-->
+
+- Examples:
```

Comment:
What do you think about also enclosing lines 33-38 in comment brackets, to avoid the user accidentally including this in their new article?